static/src/javascripts/projects/common/modules/experiments/tests/video-series-page.js

```diff
@@ -0,0 +1,36 @@
+define([
+    'common/utils/config'
+], function (
+    config
+) {
+    return function () {
+
+        this.id = 'VideoSeries';
+        this.start = '2016-04-21';
+        this.expiry = '2016-04-25';
+        this.author = 'James Gorrie';
+        this.description = 'New video series page (initial numbers test)';
+        this.audience = 1.0;
+        this.audienceOffset = 0.0;
+        this.successMeasure = '';
```

Comment:
Sounds good
On 21 Apr 2016 5:48 p.m., "James Gorrie" <notifications@github.com> wrote:

> In
> static/src/javascripts/projects/common/modules/experiments/tests/video-series-page.js
> <https://github.com/guardian/frontend/pull/12638#discussion_r60615997>:
>
> > @@ -0,0 +1,36 @@
> > +define([
> > +    'common/utils/config'
> > +], function (
> > +    config
> > +) {
> > +    return function () {
> > +
> > +        this.id = 'VideoSeries';
> > +        this.start = '2016-04-21';
> > +        this.expiry = '2016-04-25';
> > +        this.author = 'James Gorrie';
> > +        this.description = 'New video series page (initial numbers test)';
> > +        this.audience = 1.0;
> > +        this.audienceOffset = 0.0;
> > +        this.successMeasure = '';
>
> To have them there - I can add them in when we put the actual test in
> (which will be using the same files)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/12638/files/c2d03949a85288ea5f843b075f028371a702b49c#r60615997>
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396